ktxspec.adoc

```diff
@@ -2516,6 +2520,8 @@ include::appendices/vendor-metadata.adoc[]
                                       files.
                                     - Add rules for use of transfer
                                       functions.
+                                    - Forbid used of
```

Comment:
s/used/use

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/224/commits/a4834ebeafb59cf8902bd1e93305c555d79863b3.